sw/device/tests/rstmgr_alert_info_test.c

```diff
@@ -635,7 +646,7 @@ bool test_main(void) {
   LOG_INFO("reset info = 0x%02X", rst_info);
   global_alert_called = 0;
 
-  if (rst_info == kDifRstmgrResetInfoPor) {
+  if (rst_info & kDifRstmgrResetInfoPor) {
```

Comment:
Can you add a TODO to change it back once #13098 is resolved? Under those conditions, rstmgr may have a bug if this test shows other reset sources than power-on reset.

Reply:
Sure #13277 (the PR blocked by this issue) should resolve that issue.

Reply:
Done.

Reply:
Thanks!